<launch>

    <arg name="base_dir" default="$(find ros_pub_and_sub_rgbd_and_cloud)" />
    <arg name="config_file" default="config/rgbd_pub_config.yaml" />

    <node pkg="tf" type="static_transform_publisher"
        name="set_camera_pose_at_origin"
        args="0 0 0 0 0 0 1 map head_camera 10"
    />

    <node pkg="ros_pub_and_sub_rgbd_and_cloud"
        type="pub_rgbd_and_cloud.py"
        name="pub_rgbd_and_cloud"
        args="--base_dir $(arg base_dir) --config_file $(arg config_file)"
        output = "screen"
    />
    
    <node pkg="rviz" type="rviz"
        name="display_rgbd_and_cloud" 
        args="-d $(find ros_pub_and_sub_rgbd_and_cloud)/config/rgbd_and_cloud.rviz" />

</launch>
